libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -456,3 +468,31 @@ export function bucketModifyItems(
         items: items.map((bucketItem: IAttributeOrMeasure): IAttributeOrMeasure => modifications(bucketItem)),
     };
 }
+
+/**
+ * Creates a new bucket by modifying items of the provided input bucket.
+ * Array of item from the input bucket will be dispatched to the modification function
+ * and the result of the modification will be included in the new bucket.
+ *
+ *
+ * @param bucket - bucket in which all items are applied the modification function
+ * @param modifications - the modification to apply to reduce the bucket items
```

Comment:
few naming things. `BucketItemReductions` sounds awkward since it is a single reducer function doing the 'reduce' operation. Perhaps better call it `BucketItemReducer` + rename the `modifications` to `reducer`

Reply:
Done